src/app/devices/views/Devices.tsx

```diff
@@ -1,30 +0,0 @@
-import { Route, Switch } from "react-router-dom";
-
-import DeviceDetails from "./DeviceDetails";
-import DeviceList from "./DeviceList";
-
-import NotFound from "app/base/views/NotFound";
-import devicesURLs from "app/devices/urls";
-
-const Devices = (): JSX.Element => {
-  return (
-    <Switch>
```

Comment:
These intermediary components are no longer required as the routes would be moved to the top level `Routes` file.